docs/source/markdown/podman-artifact-extract.1.md

```diff
@@ -0,0 +1,80 @@
+% podman-artifact-extract 1
+
+
+## WARNING: Experimental command
+*This command is considered experimental and still in development. Inputs, options, and outputs are all
+subject to change.*
+
+## NAME
+podman\-artifact\-extract - Extract an OCI artifact to a local path
+
+## SYNOPSIS
+**podman artifact extract** *artifact* *target*
+
+## DESCRIPTION
+
+Extract the blobs of an OCI artifact to a local file or directory. If the target
```

Comment:
Consider a paragraph break after the first sentence; then there would be a paragraph for files, and a paragraph for directories.